pkgs/top-level/all-packages.nix

```diff
@@ -15113,8 +15113,6 @@ in
 
   exim = callPackage ../servers/mail/exim { };
 
-  facette = callPackage ../servers/monitoring/facette { };
```

Comment:
can you add a `throw` to `aliases.nix`.
```
facette = throw "facette has been removed";
```

etc etc.

Reply:
Done